src/form-question-types.rst

```diff
@@ -1655,33 +1655,86 @@ In the parameters column, write ``max-pixels=`` followed by the desired maximum
 Audio widget
 ----------------
 
-Records audio from an external app.
+type
+  :tc:`audio`
+appearance
+  :tc:`none`
 
-.. note::
+Records audio using the device's microphone.
 
-  Some Android devices provide a default application for audio recording.
-  Others do not,
-  and the user will need to install an audio recording app.
+.. image:: /img/form-widgets/audio-start.*
+  :alt: The Audio form widget as displayed in the ODK Collect App on an Android phone. The question text is "What does it sound like?" There are two buttons: Record Sound and Choose Sound.
 
```

Comment:
It's not obvious from these docs that I can record while answering questions. I think a couple of sentences on that, a link to the field-list docs and maybe even a screenshot would be really helpful.

Reply:
Some other things to add:
- description of behavior while recording (can't leave question screen in any way)
- behavior of pause vs. stop

Reply:
Yeah all sounds good! I can look at adding this in.

Reply:
Consolidated the rest of these bullets in https://github.com/getodk/docs/pull/1280#issuecomment-740836339